README.md

```diff
@@ -170,6 +180,15 @@ $ cp config.new.js config.js
 $ node run.js
 ```
 
+### Troubleshooting
+
+You may need to install dependencies before running the application, to do
```

Comment:
I think we need to add this in the line above run.js. This will ALWAYS be needed.